packages/@react-spectrum/s2/src/Button.tsx

```diff
@@ -161,13 +178,18 @@ const button = style<ButtonRenderProps & ButtonStyleProps>({
         isDisabled: 'transparent'
       }
     },
+    variant: {
```

Comment:
the variant should probably be moved up into the fillStyle with all the other variants

Reply:
disabled styling is not working
![Screenshot 2024-12-18 at 8 40 03 am](https://github.com/user-attachments/assets/ea86fd05-f98e-4905-a01b-eb266b322d46)
because variant is winning over the fillStyle